mindsdb/integrations/handlers/youtube_handler/youtube_tables.py

```diff
@@ -148,3 +146,68 @@ def call_youtube_comments_api(self,video_id):
 
 
         return all_youtube_comments_df
+
+class YoutubeVideoTable(APITable):
+
+    """Youtube Video info  by video id Table implementation"""
+
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        select_statement_parser = SELECTQueryParser(
+            query,
+            'video',
+            self.get_columns()
+        )
+
+        selected_columns, where_conditions, order_by_conditions, result_limit = select_statement_parser.parse_query()
+
+        video_id = None
+        for op, arg1, arg2 in where_conditions:
+            if arg1 == 'video_id':
+                if op == '=':
+                    video_id = arg2
+                    break
+                else:
+                    raise NotImplementedError("Only '=' operator is supported for video_id column.")
+
+        if not video_id:
```

Comment:
Is it possible to include other columns in the `WHERE` clause? Let's try to accomplish this, however, if there is absolutely no way to do it, let's mention it clearly in the README.

Reply:
Yes, video id is needed. I have updated the readme.